8.x/core-concepts/validation.md

```diff
@@ -2,4 +2,393 @@
 
 [[toc]]
 
-Should be filled.
\ No newline at end of file
+This document talks about checking the validity of requests that are sent from the client side to the server. The Larammerce platform is written based on the Laravel framework, so it uses the same validation rules with some modifications to make it easier.
```

Comment:
This document talks about checking the validity of requests sent from the client to the server.

delete-> that are
write-> client-side/ server-side OR client/server